src/marqo/core/structured_vespa_index/structured_vespa_index.py

```diff
@@ -640,8 +794,11 @@ def _get_select_attributes(self, marqo_query: MarqoQuery) -> str:
         else:
             return '*'
 
-    def _get_score_modifiers(self, marqo_query: MarqoQuery) -> \
-            Optional[Dict[str, Dict[str, float]]]:
+    def _get_score_modifiers(self, marqo_query: MarqoQuery) -> Optional[Dict[str, Dict[str, float]]]:
```

Comment:
Will we ever passing a HybridQuery object to this method?

Reply:
No, just made a separate method for it: `_get_hybrid_score_modifiers`